python/ray/air/_internal/usage.py

```diff
@@ -115,3 +118,40 @@ def tag_scheduler(scheduler: "TrialScheduler"):
     assert isinstance(scheduler, TrialScheduler)
     scheduler_name = _find_class_name(scheduler, "ray.tune.schedulers", TUNE_SCHEDULERS)
     record_extra_usage_tag(TagKey.TUNE_SCHEDULER, scheduler_name)
+
+
+def tag_multinode_cluster_storage(
+    local_path: str, remote_path: Optional[str], sync_config: "SyncConfig"
+) -> bool:
+    import ray
+
+    if len(ray.nodes()) <= 1:
+        return False
+
+    nfs = local_path and not remote_path and os.path.ismount(local_path)
+    local = local_path and not remote_path
+
+    if nfs:
+        record_extra_usage_tag(TagKey.AIR_MULTINODE_CLUSTER_STORAGE, "nfs")
+    elif local:
+        if sync_config.syncer is None:
+            record_extra_usage_tag(
+                TagKey.AIR_MULTINODE_CLUSTER_STORAGE, "local+no_sync"
+            )
```

Comment:
Actually when does this happen in practice? Would this actually be used for NFS?

Reply:
I wanted to track users disabling syncing -- which could be indicative of syncing being too much of a bottleneck for some people. In this scenario, the user has some access to the filesystems of each node and don't need everything to get centralized to one node.

Maybe it's easiest to just think about 4 categories to track for multinode clusters:
1. "head node syncing" = "local+sync"
2. "cloud"
3. "nfs"
4. "hdfs"

Then, for all clusters (including single node if we eventually report stuff from there), have another tag that is "cloud provider":
1. "s3"
2. "gs"
3. "other"